integration-tests/tests/common/mod.rs

```diff
@@ -67,7 +67,7 @@ impl IntegrationTest {
         }
     }
 
-    pub fn run(self, timeout: u64, exit_status: i32) {
+    pub fn run(self, timeout: u64, exit_status: i32, stdout: &str) {
```

Comment:
```suggestion
    pub fn run(self, timeout: u64, exit_status: i32, stdout: impl AsRef<[u8]>) {
```
The output from the `Command` below returns a `Vec<u8>`. This can be converted to a slice (`[u8]`) trivially. Therefore, we want the input to the test to be any type that can also be trivially converted to a slice. The good news is that this **includes** `&str`! So you can still just pass a `&str` to this function. The end result is that you don't have to convert the output slice to a string. See below.